cobbler/modules/managers/import_signatures.py

```diff
@@ -119,7 +119,15 @@ def get_file_lines(self, filename: str) -> Union[List[str], List[bytes]]:
                     return file_fd.readlines()
             except Exception:
                 pass
-        if ftype.mime_type == "application/x-ms-wim":  # type: ignore
+        if (
+            ftype.mime_type == "application/x-ms-wim"  # type: ignore
+            or filename.lower().endswith(".wim")
+            and (
+                self.file_version[0] < 5
+                or self.file_version[0] == 5
```

Comment:
```suggestion
                self.file_version[0] <= 5
```

Reply:
I would agree if this condition ended there. But what follows next is the “and” operation. For example, with version 4.38, in this case we will get the wrong result.

Reply:
I see below you already agreed that with value comparisons of List/Tuple, we can increase the code readability so I'll mark this as resolved.